packages/sdks-tests/src/e2e-tests/custom-breakpoints.spec.ts

```diff
@@ -114,8 +114,7 @@ breakpoints: {
       }
     });
 
-    test('extra small mobile size', async ({ page, sdk }) => {
-      test.skip(excludeGen2(sdk));
+    test('extra small mobile size', async ({ page }) => {
```

Comment:
Seems like this is failing for the React Native SDK. Can you spend a bit of time investigating the root cause?

tips:
- look for `useTarget()` hooks: they customize code per-SDK, so check for `reactNative:` cases that do something differently
- look for any `overrides/react-native/src/*` files that may be impacting styles
- if you're not able to easily find the root cause, we can release `xsmall` breakpoint to all gen2 except RN SDK, and work on that later

Reply:
It seems custom breakpoint inline styles don't apply for react native. 
https://github.com/BuilderIO/builder/blob/main/packages/sdks/src/components/block/components/block-styles.lite.tsx#L116

I have updated the test cases accordingly.